File: BOJ/Base/Ex2869.java
/**
 *  2021-02-01
 */

import java.util.Scanner;

public class Ex2869 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int A = sc.nextInt();
		int B = sc.nextInt();
		int V = sc.nextInt();
		
		int day = (V-B) / (A-B);
		
		if((V-B) % (A-B) != 0) {
			day++;
		}
		
	
		System.out.println("달팽이가 올라가는데 걸린 날은 "+ day + "일 입니다.");
	}

}

File: BOJ/Base/Ex2869.kt
package math

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader
import java.util.*


object Main {
    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val br = BufferedReader(InputStreamReader(System.`in`))
        val st = StringTokenizer(br.readLine(), " ")
        val A = st.nextToken().toInt()
        val B = st.nextToken().toInt()
        val V = st.nextToken().toInt()
        var day = (V - B) / (A - B)
        if ((V - B) % (A - B) != 0) {
            day++
        }
        println(day)
    }
}
